modules/README

```diff
@@ -1,11 +1,36 @@
-============================================
-Chapel Modules Directory Structure and Files
-============================================
+==================================
+Chapel Modules Directory Structure
+==================================
 
 The directory structure of the modules/ directory is as follows:
 
 ./
-  internal/ : internal modules - used to implement the language
-  standard/ : standard library modules
-  layouts/  : standard domain maps for one locale
-  dists/    : standard domain maps for distributions
+
+  dists/    : standard domain maps defining distributions that target
+              multiple locales.
+
+  internal/ : Chapel code used to implement the language; modules here
+              should be code that the user either need not be aware
+              of, or would considere to be part of the Chapel language
+              itself (i.e., something you'd document in the language
+              specification rather than as part of a library's
+              documentation).
+
+  layouts/  : standard domain maps for memory layouts targeting a
+              single locale.
+
+  minimal/  : a minimal set of modules designed for developer use when
```

Comment:
Good catch!  Discontent with what was here before, I dove in headfirst, forgetting my original task.